### PR TITLE
Replace alluxio.worker.memory.size with alluxio.worker.ramdisk.size

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: bin/alluxio-mount.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ USAGE="Usage: alluxio-mount.sh [Mount|SudoMount|Umount|SudoUmount] [MACHINE]
 function init_env() {
   local libexec_dir=${ALLUXIO_LIBEXEC_DIR:-"${BIN}"/../libexec}
   . ${libexec_dir}/alluxio-config.sh
-  MEM_SIZE=$(${BIN}/alluxio getConf --unit B alluxio.worker.memory.size)
+  MEM_SIZE=$(${BIN}/alluxio getConf --unit B alluxio.worker.ramdisk.size)
```
Comment (apc999) [inline bin/alluxio-mount.sh:26]: MEM_SIZE -> RAMDISK_SIZE ?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 2]
```diff
@@ -2639,7 +2639,7 @@ public String toString() {
           .setScope(Scope.WORKER)
           .build();
   public static final PropertyKey WORKER_MEMORY_SIZE =
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:2641]: WORKER_RAMDISK_SIZE

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 3]
```diff
@@ -2639,7 +2639,7 @@ public String toString() {
...
-      new Builder(Name.WORKER_MEMORY_SIZE)
+      new Builder(Name.WORKER_RAMDISK_SIZE)
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:2642]: setAlias to WORKER_MEMORY_SIZE

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 4]
```diff
@@ -5201,7 +5201,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String WORKER_KEYTAB_FILE = "alluxio.worker.keytab.file";
     public static final String WORKER_MASTER_CONNECT_RETRY_TIMEOUT =
         "alluxio.worker.master.connect.retry.timeout";
-    public static final String WORKER_MEMORY_SIZE = "alluxio.worker.memory.size";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:5204]: Update alluxio.conf.RemovedKey#REMOVED_KEYS

Comment (yuzhu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:5204]: is it still a removed key if we have an alias for it?

**Comment by yuzhu** [comment]
@calvinjia @apc999 addressed comments, PTAL. Decided alias is probably the way to go for backcompat. Not removing the key

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 4 above)
Comment (apc999) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:5204]: yes we still put them in `RemovedKey`, just check it out and you will find examples

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 5]
```diff
@@ -5201,7 +5202,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
...
+    public static final String WORKER_MEM_SIZE = "alluxio.worker.memory.size";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:5205]: nit: WORKER_MEM_SIZE -> WORKER_MEMORY_SIZE to keep consistent with our naming convention

File: libexec/alluxio-config.sh [hunk 6]
```diff
@@ -84,7 +84,7 @@ elif [[ -n "${ALLUXIO_UNDERFS_ADDRESS}" ]]; then
 fi
 
 if [[ -n "${ALLUXIO_WORKER_MEMORY_SIZE}" ]]; then
```
Comment (apc999) [inline libexec/alluxio-config.sh:86]: ALLUXIO_WORKER_MEMORY_SIZE -> ALLUXIO_WORKER_RAMDISK_SIZE, or maybe we should keep both env var until 3.0 @calvinjia , what's our deprecation policy?

File: libexec/alluxio-config.sh [hunk 7]
```diff
@@ -84,7 +84,7 @@ elif [[ -n "${ALLUXIO_UNDERFS_ADDRESS}" ]]; then
...
-  ALLUXIO_JAVA_OPTS+=" -Dalluxio.worker.memory.size=${ALLUXIO_WORKER_MEMORY_SIZE}"
+  ALLUXIO_JAVA_OPTS+=" -Dalluxio.worker.ramdisk.size=${ALLUXIO_WORKER_MEMORY_SIZE}"
```
Comment (apc999) [inline libexec/alluxio-config.sh:87]: ALLUXIO_WORKER_MEMORY_SIZE -> ALLUXIO_WORKER_RAMDISK_SIZE

File: core/common/src/main/java/alluxio/conf/RemovedKey.java [hunk 8]
```diff
@@ -132,6 +133,7 @@ private static String replacedSince(String version, String newProperty) {
       put("alluxio.worker.file.persist.pool.size", removedSince(V2_1_0));
       put("alluxio.worker.file.persist.rate.limit", removedSince(V2_1_0));
       put("alluxio.worker.file.persist.rate.limit.enabled", removedSince(V2_1_0));
+      put("alluxio.worker.memory.size", removedSince(V2_4_0));
```
Comment (apc999) [inline core/common/src/main/java/alluxio/conf/RemovedKey.java:136]: `replacedSince(V2_4_0,  PropertyKey.Name.WORKER_RAMDISK_SIZE)`

File: libexec/alluxio-config.sh (hunk 6 above)
Comment (apc999) [reply libexec/alluxio-config.sh:86]: @yuzhu ping

File: libexec/alluxio-config.sh (hunk 7 above)
Comment (apc999) [reply libexec/alluxio-config.sh:87]: @yuzhu

File: core/common/src/main/java/alluxio/conf/RemovedKey.java (hunk 8 above)
Comment (yuzhu) [reply core/common/src/main/java/alluxio/conf/RemovedKey.java]: fixed

File: libexec/alluxio-config.sh (hunk 6 above)
Comment (yuzhu) [reply libexec/alluxio-config.sh:86]: done

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**